### DISCUSSION TITLE
Several Ideas how to improve UX

### DISCUSSION THREAD

**Discussion by zarb1n** [thesamim/TickTickSync#80] [Ideas]
Hello thesamim,
 
First off, I want to express my immense appreciation for your addon — it's truly phenomenal, and I thoroughly enjoy using it. Thank you immensely for your hard work!
 
As a committed user of the addon, I've come across a few minor inconveniences, and I'd like to offer my insights in the hopes that they may be of some assistance to you:

1. Regarding the [issue](https://github.com/thesamim/TickTickSync/issues/15) with the handling of the "Won't do" status, it might be beneficial to provide users with the option to choose the markdown syntax for indicating this status within square brackets. For instance, I personally prefer `[-]` for such tasks, while others might prefer `[/]` or some other variation.

2. Personally, I prefer to avoid any additional tags (including `#ticktick`), so it would be ideal for me to utilize the Full Vault Sync feature while also having the option to prevent TickTick checkboxes from being converted into full-fledged tasks. I'm content with TickTick tasks' content (including checkboxes) not being displayed in Obsidian.

3. I'd like to suggest a minor enhancement to the user experience by allowing for the selection of multiple tags and projects to be synchronized. For instance, I've been using TickTick for around four years now, and I have numerous archived lists and collaborative lists that I'd like to exclude (or include others). While I understand that I can achieve this by assigning specific tags to the tasks I wish to sync and configuring this in TickTickSync settings, it does introduce an additional tag, which could be cumbersome.

4. Additionally, it might be beneficial to utilize the "scheduled" date for the start of a task if the TickTick task includes both start and end dates (interval mode). This would provide more comprehensive information, as currently, in Obsidian, I'm only able to see when a task ends. For instance, in the case of a meeting task, relying solely on Obsidian could potentially lead to me being late, as I wouldn't know when the task starts. Also, when I create an interval task in TickTick, the addon edits the time of the task.

Once again, thank you tremendously for the plugin.
 
P.S. I've consolidated all my feedback into this single message. If this format is bothersome or if you prefer feedback to be divided into separate topics, please let me know, and I'll adjust accordingly in future communications.

**Comment by thesamim** [comment]
@zarb1n : Thanks for the feedback.

Will consider all your suggestions in due time.
